languages/csharp/exercises/concept/numbers-signedness/.docs/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+Number types get a fixed amount of memory assigned, therefore they can represent a fixed range of numbers. A signed type can represent both positive and negative numbers, unsigned types can only represent positive numbers. A unsigned type doesn't need a byte to represent if a number is positive or negative so it can use that byte to represent a bigger range of positive numbers.
```

Comment:
```suggestion
Number types get a fixed amount of memory assigned, therefore they can represent a fixed range of numbers. A signed type can represent both positive and negative numbers, unsigned types can only represent positive numbers. A unsigned type doesn't need a bit to represent if a number is positive or negative so it can use that bit to represent a bigger range of positive numbers.
```

Reply:
Oh good catch, always mix those up. ;-)